Date: 2014-09-12 12:01:17.376375609 +0200
From: Jan Engelhardt <jengelh@inai.de>

build: resolve bogus installation of hfst_foma causing trouble

hsft_foma is not a script, and it is installed by
back-ends/foma/Makefile.am already. Do not install it twice.

---
 scripts/Makefile.am |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: hfst-3.12.2/scripts/Makefile.am
===================================================================
--- hfst-3.12.2.orig/scripts/Makefile.am
+++ hfst-3.12.2/scripts/Makefile.am
@@ -16,9 +16,9 @@
 ## along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 if GENERATE_FOMA_WRAPPER
-bin_SCRIPTS = hfst-foma-wrapper.sh hfst-foma ${top_srcdir}/back-ends/foma/hfst_foma$(EXEEXT)
+bin_SCRIPTS = hfst-foma-wrapper.sh hfst-foma
 EXTRA_DIST=hfst-foma generate-cc-files.sh
-AM_INSTALLCHECK_STD_OPTIONS_EXEMPT=hfst-foma-wrapper.sh hfst-foma ${top_srcdir}/back-ends/foma/hfst_foma$(EXEEXT)
+AM_INSTALLCHECK_STD_OPTIONS_EXEMPT=hfst-foma-wrapper.sh hfst-foma
 else
 EXTRA_DIST=generate-cc-files.sh
 endif
